README.md

```diff
@@ -68,13 +68,13 @@ For API documentation we use [Swagger](https://cans.cachet.dk/dev/swagger-ui/ind
 
 # Deployment
 
-1. Edit the [.env file](.env). Choose a profile from the list of [Profiles](#profiles).
+1. Edit the [.env file](.env.local). Choose a profile from the list of [Profiles](#profiles).
```

Comment:
What is the point of changing this in the Readme?

Reply:
my version issues, shouldn't be in develop